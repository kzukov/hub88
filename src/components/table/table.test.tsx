import React from "react";
import Table, { TableColumn, TableData } from "./table";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";

describe("<Table />", () => {
  test("matches snapshot", () => {
    const columns: TableColumn[] = [
      {
        key: "test",
        content: "name",
      },
    ];

    const data: TableData[] = [
      {
        test: "value",
      },
    ];

    const { asFragment } = render(<Table columns={columns} data={data} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
