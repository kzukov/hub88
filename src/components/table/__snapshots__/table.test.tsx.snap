// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> matches snapshot 1`] = `
<DocumentFragment>
  <table
    class="table"
  >
    <thead>
      <tr>
        <th
          scope="col"
        >
          Index
        </th>
        <th
          scope="col"
        >
          name
        </th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <th
          scope="row"
        >
          0
        </th>
        <td>
          value
        </td>
      </tr>
    </tbody>
  </table>
</DocumentFragment>
`;
